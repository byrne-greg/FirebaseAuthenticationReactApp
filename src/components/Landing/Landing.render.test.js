import React from "react";
import { render, cleanup } from "@testing-library/react";
import Landing from ".";

afterEach(cleanup);

describe("Landing render tests", () => {
  it("page renders as expected", () => {
    // GIVEN

    // WHEN
    const page = render(<Landing />);

    // THEN
    expect(page.baseElement).toMatchSnapshot();
  });
});
