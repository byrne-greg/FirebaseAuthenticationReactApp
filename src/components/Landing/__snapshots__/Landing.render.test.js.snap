// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing render tests page renders as expected 1`] = `
<body>
  <div>
    <div>
      <h1>
        Landing
      </h1>
    </div>
  </div>
</body>
`;
